tailwind.config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   separator: '-',
   important: false,
   purge: {
-    enabled: true,
+    enabled: false,
```

Comment:
This needs to be true before we can merge, and you'll have to re-run a yarn prod after setting back to true.

Reply:
If we change it to true, my code doesn't work. Perhaps we can chat about this and figure out what to do.

There is a snippet of javascript that applies a TW class, but because we never load the TW class before that code fires, it gets stripped out of the CSS i think and doesn't do anything when applied.